FAQ.md

```diff
@@ -105,31 +105,54 @@ But you can also use a crate like [ctrlc](https://docs.rs/ctrlc/latest/ctrlc/).
 ## How to use `log` or `tracing` as default log backend
 
 * **log**, add the feature flag `logger_log` to the dependency in `Cargo.toml`
-    ```toml
-    iceoryx2 = { version = "0.1.0", features = ["logger_log"]}
-    ```
+  ```toml
+  iceoryx2 = { version = "0.1.0", features = ["logger_log"]}
+  ```
 * **tracing**, add the feature flag `logger_tracing` to the dependency in
   `Cargo.toml`
-    ```toml
-     iceoryx2 = { version = "0.1.0", features = ["logger_tracing"]}
-    ```
+  ```toml
+   iceoryx2 = { version = "0.1.0", features = ["logger_tracing"]}
+  ```
+
+## Supported log levels
+
+iceoryx2 support different log levels
+`trace, debug, info, warning, error, fatal`
 
 ## How to set the log level
 
+While developing, iceoryx2 gets its default logging level from `.cargo/config.toml`, 
+but this can be over-ridden by using environment variable `IOX2_LOG_LEVEL` that can
+be set with one of the supported levels.
+
+`export IOX2_LOG_LEVEL=Trace`
+
+then in the `main.rs` call the function to read the environment variable
+
 ```rust
 use iceoryx2::prelude::*
 
 // ...
 
+// Reads LogLevel from env and defaults to LogLevel INFO
+// if the environemnt variable has an unsupported value
```

Comment:
Small typo. Also, if the env variable is not set, the default log level will be used.
```suggestion
// if the environment variable is not set or has an unsupported value
```